samza-core/src/main/java/org/apache/samza/storage/TaskStorageCommitManager.java

```diff
@@ -181,27 +176,26 @@ public void persistToLocalFileSystem(Checkpoint checkpoint) {
   }
 
   /**
-   * // TODO MED dchen: fix documentation
-   * Cleanup the commit state for each of the task backup managers
+   * Cleanup on the commit state from the {@link #commit(CheckpointId)} call. Evokes the cleanup the commit state
```

Comment:
s/Evokes/Invokes in this class.

Reply:
"Performs any post-commit and cleanup actions after the {@link Checkpoint} is successfully written to the checkpoint topic. Invokes {@link TaskStorageBackupManager#cleanup} on each of the configured backup managers. Deletes all local store checkpoint directories older than the {@param latestCheckpointId}